autogpt_platform/docker-compose.yml

```diff
@@ -137,29 +141,31 @@ services:
     extends:
       file: ./supabase/docker/docker-compose.yml
       service: db
+  
+  # Required for db
   vector:
     <<: *supabase-services
     extends:
       file: ./supabase/docker/docker-compose.yml
       service: vector
 
-  deps:
-    <<: *supabase-services
-    profiles:
-      - local
-    image: busybox
-    command: /bin/true
-    depends_on:
-      - studio
-      - kong
-      - auth
-      - rest
-      - realtime
-      - storage
-      - imgproxy
-      - meta
-      - functions
-      - analytics
-      - db
-      - vector
-      - redis
+  # deps:
+  #   <<: *supabase-services
```

Comment:
I need this for local development, this is the images need to be run except the application services, can we adjust this accordingly instead of commenting ?

Reply:
Currently in dev, it only requires the containers I have left uncommented. 
Which of the additional services do you need that have been removed?

This specifically doesn't really do anything other than exiting with success when the deps have started.